### PR TITLE
Upped dubbd to 0.12.1

### PR DESCRIPTION
**Pull request by Michael-Kruggel**

### CODE REVIEW

**Review by blancharda — CHANGES_REQUESTED** [review]
Let's chat about upgrade paths..
I think we're going to need an 0.11.* release as well.

File: Makefile [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-ZARF_VERSION := v0.29.2
+ZARF_VERSION := v0.30.1
 
 # The version of the build harness container to use
 BUILD_HARNESS_REPO := ghcr.io/defenseunicorns/build-harness/build-harness
 # renovate: datasource=docker depName=ghcr.io/defenseunicorns/build-harness/build-harness
 BUILD_HARNESS_VERSION := 1.10.2
 # renovate: datasource=docker depName=ghcr.io/defenseunicorns/packages/dubbd-k3d extractVersion=^(?<version>\d+\.\d+\.\d+)
-DUBBD_K3D_VERSION := 0.10.1
+DUBBD_K3D_VERSION := 0.12.0
```
[the remark this answers is not in the document]
Comment (blancharda) [reply Makefile:11]: I think I've been talked off the cliff..

**Review by corang — APPROVED** [review]

**Pull request merged**